pages/validator/external-chains/optimism.md

```diff
@@ -0,0 +1,440 @@
+# Optimism
+-----------
+
+import Markdown from 'markdown-to-jsx'
+import Tabs from '../../../components/tabs'
+import CodeBlock from '../../../components/code-block'
+import { NextraTab, NextraTabs } from 'nextra-theme-docs'
+
+Instructions to set up your Optimism node.
+
+## Requirements
+
+- [Setup your Axelar validator](/validator/setup)
+- Minimum hardware requirements: 8 AWS vCPU+, 16GB RAM, 80GB+ free storage space.
+- MacOS or Ubuntu 20.04 (tested on 20.04)
+- [Official Documentation](https://community.optimism.io/docs/developers/build/run-a-node/#)
+
+
+## Prerequisites
+```bash
+sudo apt-get install jq -y
+snap install docker 
+```
+
+## Clone Repo
+<Tabs tabs={[
+{
+title: "Mainnet",
+content: <CodeBlock language="bash">
+{`
+git clone https://github.com/smartcontracts/simple-optimism-node.git
+cd simple-optimism-node
+cp .env.example .env
+`}
+</CodeBlock>
+},
+{
+title: "Testnet",
+content: <CodeBlock language="bash">
+{`
+git clone https://github.com/ethereum-optimism/simple-optimism-node.git
+cd simple-optimism-node
+cp .env.example .env
+`}
+</CodeBlock>
+}
+]} />
+
+## Set environment variables
+
+<Tabs tabs={[
+{
+title: "Mainnet",
+content: <CodeBlock language="bash">
+{`
+NETWORK_NAME=mainnet
+NODE_TYPE=full
+# can be l1 or l2
+SYNC_SOURCE=
+#  l2 node to check blocks against
+HEALTHCHECK__REFERENCE_RPC_PROVIDER=
+# l1 node RPC to check state roots against
+FAULT_DETECTOR__L1_RPC_PROVIDER=
+# can be l1 node or l2 node, depends on what sync source you have mentioned in the variable SYNC_SOURCE
+DATA_TRANSPORT_LAYER__RPC_ENDPOINT=
+`}
+</CodeBlock>
+},
+{
+title: "Testnet",
+content: <CodeBlock language="bash">
+{`
+# only goerli is supported for now
+NETWORK_NAME=goerli
+# can be full or archive
+NODE_TYPE=full
+# only download option is available for now
+BEDROCK_SOURCE=download
+# L1 node that the op-node will get chain data from - if you are using third party - make sure it will not get rate limited
```

Comment:
Should emphasize that this should be a validator's own ethereum/goerli RPC node, using a third party L1 node has negative security/performance impact. You can change the examples below appropriately.

Reply:
Sure, I used them in example to contrast where to use a eth node and where to use optimism node, since people can distinguish from the infura URL, but yeah we might be setting bad examples so I will just put YOUR_ETH_NODE